book/07-extending-learners.qmd

```diff
@@ -5,47 +5,53 @@ library(mlr3)
 library(mlr3book)
 ```
 
+Although many learners are already included in the mlr3verse, there might be a situation in which your algorithm of choice is not implemented.
 Here, we show how to create a custom mlr3learner step-by-step using `r ref("mlr3extralearners::create_learner")`.
-
-**It is strongly recommended** that you **first** open a [learner request issue](https://github.com/mlr-org/mlr3extralearners/issues/new?assignees=&labels=new+learner&template=learner-request-template.md&title=%5BLRNRQ%5D+Add+%3Calgorithm%3E+from+package+%3Cpackage%3E) to discuss the learner you want to implement if you plan on creating a pull request to the mlr-org. This allows us to discuss the purpose and necessity of the learner before you start to put the real work in!
+If you think the learner will be useful to others as well, you can consider creating a pull request in the [mlr3extralearners](https://github.com/mlr-org/mlr3extralearners) repository.
```

Comment:
I think this sentence can be left out, every learner has an added value to multiple people and the decision should also not be moved to potential contributors.

Reply:
agree